packages/filebrowser/src/listing.ts

```diff
@@ -2808,7 +2808,11 @@ export namespace DirListing {
             small: trans.__('Modified'),
             large: trans.__('Last Modified')
           }),
-        file_size: () => this.createHeaderItemNode(trans.__('File Size')),
+        // file_size: () => this.createHeaderItemNode(trans.__('Size')),
```

Comment:
Let's remove the commented out code:

```suggestion
```

:)

Reply:
Thank you, was hiding in there 😂

Reply:
Ready to close?